client/src/pages/dashboard/components/Map/SendFeedback/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from "react";
+import { Button, FormGroup, Label, Input } from "reactstrap";
+import { useForm } from "react-hook-form";
+import { connect, useDispatch } from "react-redux";
+import { object, string } from "prop-types";
+import { size } from "lodash";
+import { useTranslation } from "react-i18next";
+
+// eslint-disable-next-line import/no-unresolved
+import senMapFeedback from "store/sendMapFeedback/actions";
+import Places from "../Places";
+import { FormStyle } from "./style";
+
+const SendFeedback = ({ id, label, locationData }) => {
+  const { t } = useTranslation();
+
+  const { register, handleSubmit } = useForm();
+  const dispatch = useDispatch();
+
+  const [texVal, setTextVal] = useState("");
+  const [isTexVal, setIsTextVal] = useState(true);
+  const [isButton, setIsButton] = useState(true);
+
+  useEffect(() => {
+    if (id) setIsTextVal(false);
+  }, [id]);
+
+  const textAriaHandler = (e) => {
+    const { value } = e.target;
+    setTextVal(value);
+    return value && id ? setIsButton(false) : setIsButton(true);
+  };
+
+  const saveHandler = () => {
+    const dataFeedback = { id, label, desc: texVal };
+
+    if (locationData && id) {
+      dataFeedback.location = locationData;
+      dispatch(senMapFeedback(dataFeedback));
+    }
+    setTextVal("");
+    setIsTextVal(true);
+    setIsButton(true);
+  };
+
+  return (
+    <FormStyle onSubmit={handleSubmit()}>
```

Comment:
Why did you call the function `handleSubmit` right away?

Reply:
because I use react-hook-form